cpp/modmesh/view/R3DWidget.cpp

```diff
@@ -103,6 +103,20 @@ void R3DWidget::resizeEvent(QResizeEvent * event)
     m_container->resize(event->size());
 }
 
+void R3DWidget::resetCamera(Qt3DRender::QCamera * camera)
+{
+    // Set up the camera.
+    camera->lens()->setPerspectiveProjection(45.0f, 16.0f / 9.0f, 0.1f, 1000.0f);
+    camera->setPosition(QVector3D(0.0f, 0.0f, 10.0f));
```

Comment:
Is possible to determine the z-axis position of the camera based on the size of the displayed object ?
If the object is too large, the camera might too close then the entire object will not be fitted in the screen.